src/consensus/validation.h

```diff
@@ -71,19 +96,7 @@ class CValidationState {
     bool IsError() const {
         return mode == MODE_ERROR;
     }
-    bool IsInvalid(int &nDoSOut) const {
-        if (IsInvalid()) {
-            nDoSOut = nDoS;
-            return true;
-        }
-        return false;
-    }
-    bool CorruptionPossible() const {
```

Comment:
Alternatively, you could just change this function to return whether `reason` is set to `MUTATED` or `WITNESS_MUTATED`

Reply:
Then it'd be dead code and forcing the caller to be more clear about what *exactly* they're checking for is much more stable.

Reply:
Understood. This is fine as is.